src/status_im/ui2/screens/chat/composer/input.cljs

```diff
@@ -378,29 +388,30 @@
-                                         :text-input-handle (rn/find-node-handle @text-input-ref)
-                                         :full-text         full-text
-                                         :menu-items        menu-items
-                                         :chat-id           chat-id
-                                         :selection-event   selection-event})))
-              props (merge props
-                           {:ref                 ref
-                            :style               (dissoc style :margin-horizontal)
-                            :on-selection-change on-selection-change
-                            :on-selection        on-selection})]
+              on-selection        (fn [^js event]
+                                    (let [native-event           (.-nativeEvent event)
+                                          native-event           (transforms/js->clj native-event)
+                                          {:keys [eventType content selectionStart
+                                                  selectionEnd]} native-event
+                                          full-text              (:input-text
+                                                                  (rf/sub [:chats/current-chat-input]))]
+                                      (on-menu-item-touched {:first-level       first-level
+                                                             :event-type        eventType
+                                                             :content           content
+                                                             :selection-start   selectionStart
+                                                             :selection-end     selectionEnd
+                                                             :text-input        @text-input-ref
+                                                             :text-input-handle (rn/find-node-handle
+                                                                                 @text-input-ref)
+                                                             :full-text         full-text
+                                                             :menu-items        menu-items
+                                                             :chat-id           chat-id
+                                                             :selection-event   selection-event})))
+              props               (assoc props
+                                         :ref                 ref
+                                         :style               (dissoc style :margin-horizontal)
+                                         :on-selection-change on-selection-change
+                                         :on-selection        on-selection)]
           [rn/selectable-text-input {:menuItems @menu-items :style style}
-           [rn/text-input props
+           [rn/text-input (dissoc props :default-value)
```

Comment:
If `default-value` is present, the component duplicates all the text while typing